spec/features/create_work_spec.rb

```diff
@@ -135,7 +135,7 @@
     end
   end
 
-  context "with valkyrie resources", metadata_adapter: :postgres_adapter do
+  context "with valkyrie resources", valkyrie_adapter: :postgres_adapter do
```

Comment:
`metadata_adapter` was the wrong metadata tag and wasn't actually being handled.